XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1,1148 +1,1148 @@
+    <Content Include="Assets\Tiles\LargeTile.scale-125.png" />
+    <Content Include="Assets\Tiles\LargeTile.scale-150.png" />
+    <Content Include="Assets\Tiles\LargeTile.scale-200.png" />
+    <Content Include="Assets\Tiles\LargeTile.scale-400.png" />
+    <Content Include="Assets\MainPage_Banner.jpg" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.altform-unplated_targetsize-16.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.altform-unplated_targetsize-24.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.altform-unplated_targetsize-256.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.altform-unplated_targetsize-32.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.altform-unplated_targetsize-48.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.scale-100.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.scale-125.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.scale-150.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.targetsize-16.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.targetsize-24.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.targetsize-256.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.targetsize-32.png" />
+    <Content Include="Assets\Tiles\SmallTile-sdk.targetsize-48.png" />
+    <Content Include="Assets\Tiles\SmallTile.scale-100.png" />
+    <Content Include="Assets\Tiles\SmallTile.scale-125.png" />
+    <Content Include="Assets\Tiles\SmallTile.scale-150.png" />
+    <Content Include="Assets\Tiles\SmallTile.scale-200.png" />
+    <Content Include="Assets\Tiles\SmallTile.scale-400.png" />
+    <Content Include="Assets\Tiles\splash-sdk.scale-100.png" />
+    <Content Include="Assets\Tiles\splash-sdk.scale-125.png" />
+    <Content Include="Assets\Tiles\splash-sdk.scale-150.png" />
+    <Content Include="Assets\Tiles\splash-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\splash-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-100.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-125.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-150.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\squaretile-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-100.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-125.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-150.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-200.png" />
+    <Content Include="Assets\Tiles\StoreLogo-sdk.scale-400.png" />
+    <Content Include="Assets\Tiles\WideTile.scale-100.png" />
+    <Content Include="Assets\Tiles\WideTile.scale-125.png" />
+    <Content Include="Assets\
```

Comment:
Why isn't this the only line that has changed?

Reply:
Weird. Visual studio and codeflow show only one line change.

Reply:
I fixed it.